dash/testing/browser.py

```diff
@@ -504,7 +504,15 @@ def visit_and_snapshot(self, resource_path, hook_id, assert_check=True):
             if path != resource_path:
                 logger.warning("we stripped the left '/' in resource_path")
             self.driver.get("{}/{}".format(self.server_url.rstrip("/"), path))
+
+            # wait for the hook_id to present and all callbacks get fired
             self.wait_for_element_by_id(hook_id)
+            until(
+                lambda: self.redux_state_rqs
+                and all((_["responseTime"] for _ in self.redux_state_rqs)),
```

Comment:
I don't think `responseTime` is even *in* the object while it's waiting for a response? so perhaps
```suggestion
                and all((_.get("responseTime") for _ in self.redux_state_rqs)),
```